src/ip_station_points_grid_mod.F90

```diff
@@ -53,21 +53,22 @@ end subroutine init_grib2
   !> Interpolate gridded data to a series of station points.
   !>
```

Comment:
Let me further refine these definitions:

NPTS - Maximum number of points.
XPTS - X point coordinates. Always output.
YPTS - Y point coordinates. Always output.
RLAT - Point latitudes. Always input.
RLON - Point longitudes. Always input.

Reply:
OK, fixed.